src/control/controlobjectscript.cpp

```diff
@@ -57,7 +57,7 @@ void ControlObjectScript::removeScriptConnection(const ScriptConnection& conn) {
 
 void ControlObjectScript::disconnectAllConnectionsToFunction(const QScriptValue& function) {
     // Make a local copy of m_scriptConnections because items are removed within the loop.
-    QList<ScriptConnection> connections = m_scriptConnections;
+    auto connections = m_scriptConnections;
```

Comment:
Is clazy complaining here? It is OK for me to use auto here, but I think it is at the border line between readable code and clutter removal. The type of m_scriptConnections is out of sight, but if we use it directly, we have also no type visible. 
I think we should add the case to our auto use rules.

Reply:
should it be const auto by the way?

Reply:
clazy was only complaining about the use of a QList when a QVector would be more efficient (it claims), I changed it to auto so changing the type of the variable in the future doesn't require changing these lines too. I'm happy to change it back though.

Reply:
I agree here, better to make the type explicit when copying a variable.  auto is better for assignment from a function where the return type is ~clear.

Reply:
changed to const QVector